Block/Checkout/Onepage/Success/AdditionalInformation.php

```diff
@@ -0,0 +1,71 @@
+<?php
+namespace Omise\Payment\Block\Checkout\Onepage\Success;
+
+class AdditionalInformation extends \Magento\Framework\View\Element\Template
+{
+    /**
+     * @var \Magento\Checkout\Model\Session
+     */
+    protected $_checkoutSession;
+
+    /**
+     * $var \Omise\Payment\Helper\OmiseHelper
+     */
+    protected $_helper;
+
+    /**
+     * @var string
+     */
+    protected $paymentType;
+
+    /**
+     * @param \Magento\Framework\View\Element\Template\Context $context
+     * @param \Magento\Checkout\Model\Session $checkoutSession
+     * @param \Omise\Payment\Helper\OmiseHelper $helper
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Framework\View\Element\Template\Context $context,
+        \Magento\Checkout\Model\Session $checkoutSession,
+        \Omise\Payment\Helper\OmiseHelper $helper,
+        array $data = []
+    ) {
+        $this->_checkoutSession = $checkoutSession;
+        $this->_helper = $helper;
+        $this->order       = $this->getOrder();
+        $this->paymentData = $this->order->getPayment()->getData();
+        $this->paymentType = $this->getPaymentAdditionalInformation('payment_type');
+        parent::__construct($context, $data);
+    }
```

Comment:
maybe giving a code alignment would be nice (either the same vertical alignment or 1 space)

Reply:
done @ 81b078e0b967d16b53bfb532673fd627c51c857f